services/galley/src/Galley/API/Teams.hs

```diff
@@ -512,8 +512,8 @@ getLegalholdStatus (uid ::: tid ::: ct) = do
 getSSOStatusInternal :: TeamId ::: JSON -> Galley Response
 getSSOStatusInternal (tid ::: _) = do
     defConfig <- do
-        FeatureFlags flags <- view (options . optSettings . setFeatureFlags)
-        pure $ if FeatureSSO `elem` flags
+        yes <- view (options . optSettings . featureEnabled FeatureSSO)
+        pure $ if yes
```

Comment:
`if yes` reads a bit like `if true`, but it's sometimes true and sometimes not, so it's not a good variable name. Suggestion: Rename `yes` to `featureSSO`.